src/main/java/de/zalando/aruha/nakadi/service/StrategiesRegistry.java

```diff
@@ -9,20 +9,13 @@
 public class StrategiesRegistry {
 
     public static final PartitionStrategyDescriptor HASH_PARTITION_STRATEGY =
-            new PartitionStrategyDescriptor("hash", "This strategy will use the event " +
```

Comment:
Another issue, when keeping the documentation in the API is where to keep it?

* `GET /registry/partition-strategies`
* Definition of `eventtype.partition_strategy`
* Definition of `EventPartitionStrategy`

I really think, it is not the right place to keep it.